src/server/mod.rs

```diff
@@ -117,6 +108,32 @@ impl BlockingRequestAction for InitializeRequest {
             ));
         }
 
+        if !dups.is_empty() {
+            let mut msg = String::new();
+            for kv in &dups {
+                msg += &format!("{}: ", kv.0);
+                for v in kv.1 {
+                    msg += &format!("{}, ", v);
+                }
+                msg += "; ";
+            }
+            out.notify(Notification::<ShowMessage>::new(ShowMessageParams {
+                typ: MessageType::Warning,
+                message: format!("duplicated option: {}", msg.clone()),
```

Comment:
Perhaps we could be clearer with the message, maybe instead of _"option"_ we call it _"RLS configurtion"_

Reply:
Ok